### PR TITLE
Add authors to list of exposed fields

### PR DESCRIPTION
**Pull request by laugri**
Hi @conradj , this PR adds the `authors` field to the GraphQL API.

### CODE REVIEW

**Comment by conradj** [comment]
Thanks a lot @nakleiderer - could you update the example query with your changes on `README.md`?

Finally, can you bump the version number in package.json? This will automatically publish to npm on merge 🚀

**Comment by laugri** [comment]
Hey, I noticed my code is plain wrong anyway so I'll push some fixes before that.

**Comment by laugri** [comment]
Ok @conradj , this one should be ok now :)

File: gatsby-node.js [hunk 1]
```diff
@@ -114,6 +114,24 @@ exports.sourceNodes = async ({ actions }, pluginOptions) => {
         ? new URL(datum.resolved_url).hostname
         : "";
 
+    const authors = datum.authors
+      ? Object.values(datum.authors).map(author =>
+          createNode({
+            ...author,
+            id: author.item_id,
```
Comment (conradj) [inline gatsby-node.js:121]: @laugri if you make this `id: author.author_id` that should work I think?

**Pull request merged**